components/camera/videosource/query.go

```diff
@@ -203,14 +216,14 @@ func queryDriverProperties(
 		if isAvailable {
 			err := d.Open()
 			if err != nil {
-				logger.Debugw("error opening driver for querying", "error", err)
+				logger.Infow("error trying to open driver for querying", "error", err)
```

Comment:
these should be info logs since they include valuable information such as IO errors when access the driver for query